sdk/typescript/src/signers/signer-with-provider.ts

```diff
@@ -163,8 +163,8 @@ export abstract class SignerWithProvider implements Signer {
       sig.signature,
       sig.pubKey,
       bcs,
-      version?.major == 0 && version?.minor < 18 ? 'base64' : 'base58',
-      version?.major == 0 && version?.minor < 18 ? false : true
+      'base58',
+      true
```

Comment:
ditto: we should remove  the version on line 149 and both parameters in https://github.com/MystenLabs/sui/blob/735bd4d0bd867bf7df9ae35c1d4c114bb9f68787/sdk/typescript/src/types/common.ts#L142-L143